dist/bin/current/helperScripts/compileRoddyPlugin.sh

```diff
@@ -4,13 +4,13 @@ set -e
 increasebuildonly=${increasebuildonly-false}
 
 # Check for configured plugin directories. Set empty if no dirs are configured.
-pluginDirectories=`grep pluginDirectories ${customconfigfile}`
-[[ -z "$pluginDirectories" ]] && pluginDirectories="pluginDirectories="
+pluginDirectories=`grepFromConfigFile pluginDirectories $customconfigfile`
```

Comment:
I changed the line and in the course of editing removed them, apparently. As they are meaningless in this context I could leave them away, but if you wish I add them again.